cluster/gce/gci/configure-helper.sh

```diff
@@ -1736,29 +1736,11 @@ function start-kube-apiserver {
     container_env="\"env\":[{${container_env}}],"
   fi
 
-  if [[ -n "${ETCD_KMS_KEY_ID:-}" ]]; then
-    ENCRYPTION_PROVIDER_CONFIG=$(cat << EOM | base64 | tr -d '\r\n'
-kind: EncryptionConfig
-apiVersion: v1
-resources:
-  - resources:
-    - secrets
-    providers:
-    - kms:
-       name: grpc-kms-provider
-       cachesize: 1000
-       endpoint: unix:///var/run/kmsplugin/socket.sock
-EOM
-)
-  fi
+  src_file="${src_dir}/kube-apiserver.manifest"
```

Comment:
make local?

Reply:
It is used in other places (ex. start-etcd-empty-dir-clean-up-pod).

Reply:
start-etcd-empty-dir-clean-up-pod declares src_file as local. Shouldn't we do the same here?